runtime/descriptor.cpp

```diff
@@ -172,9 +175,9 @@ std::size_t DescriptorAddendum::SizeInBytes() const {
   return SizeInBytes(LenParameters());
 }
 
-std::ostream &DescriptorAddendum::Dump(std::ostream &o) const {
-  o << "  derivedType @ 0x" << std::hex
-    << reinterpret_cast<std::intptr_t>(derivedType_) << std::dec << '\n';
+llvm::raw_ostream &DescriptorAddendum::Dump(llvm::raw_ostream &o) const {
+  o << "  derivedType @ 0x"; o.llvm::raw_ostream::write_hex(
```

Comment:
You don't need the namespace and class name here.
Also maybe needs a clang-format pass.